plugins/postmortem-reminder/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/gitpod-io/gitbot/postmortem-reminder
```

Comment:
Don't forget to run `go mod tidy` :)

Reply:
BTW, it's lovely you're writing your first Prow plugin!

I feel more will follow :)

Reply:
> Don't forget to run `go mod tidy` :)

For some reason `go mod tidy` doesn't do anything for me 🤔. Well, actually it fails with:
```
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        sigs.k8s.io/controller-runtime/pkg/client imports
        k8s.io/client-go/metadata: package k8s.io/client-go/metadata provided by k8s.io/client-go at latest version v0.21.3 but not at required version v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        k8s.io/client-go/kubernetes imports
        k8s.io/client-go/discovery imports
        github.com/googleapis/gnostic/OpenAPIv2: module github.com/googleapis/gnostic@latest found (v0.5.5), but does not contain package github.com/googleapis/gnostic/OpenAPIv2
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        k8s.io/client-go/kubernetes imports
        k8s.io/client-go/kubernetes/typed/auditregistration/v1alpha1 imports
        k8s.io/api/auditregistration/v1alpha1: module k8s.io/api@latest found (v0.21.3), but does not contain package k8s.io/api/auditregistration/v1alpha1
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        k8s.io/client-go/kubernetes imports
        k8s.io/client-go/kubernetes/typed/batch/v2alpha1 imports
        k8s.io/api/batch/v2alpha1: module k8s.io/api@latest found (v0.21.3), but does not contain package k8s.io/api/batch/v2alpha1
github.com/gitpod-io/gitbot/postmortem-reminder imports
        k8s.io/test-infra/prow/flagutil imports
        k8s.io/client-go/kubernetes imports
        k8s.io/client-go/kubernetes/typed/settings/v1alpha1 imports
        k8s.io/api/settings/v1alpha1: module k8s.io/api@latest found (v0.21.3), but does not contain package k8s.io/api/settings/v1alpha1
```

But I couldn't figure out how to solve that

Reply:
You're missing the `replace` rule for k8s.io in your `go.mod`. Check out the `go.mod` of the other plugins to see what I mean